deps.windows/10-mbedtls.ps1

```diff
@@ -43,14 +39,20 @@ function Configure {
     Log-Information "Configure (${Target})"
     Set-Location $Path
 
+    if ( $ForceStatic -and $script:Shared ) {
+        $Shared = $false
+    } else {
+        $Shared = $script:Shared.isPresent
+    }
+
     $OnOff = @('OFF', 'ON')
     $Options = @(
         $CmakeOptions
-        "-DUSE_SHARED_MBEDTLS_LIBRARY=$($OnOff[$Shared.isPresent])"
-        '-DUSE_STATIC_MBEDTLS_LIBRARY=ON'
-        '-DENABLE_PROGRAMS=OFF'
-        '-DENABLE_TESTING=OFF'
-        '-DLIB_INSTALL_DIR=bin'
+        "-DUSE_SHARED_MBEDTLS_LIBRARY:BOOL=$($OnOff[$Shared])"
+        "-DUSE_STATIC_MBEDTLS_LIBRARY:BOOL=$($OnOff[$Shared -ne $true]))"
```

Comment:
no idea, i ripped the file from @PatTheMav  https://github.com/obsproject/obs-deps/pull/186
Since he managed to build fully ffmpeg on msvc, i tried to mimic what he did with mbedtls.

Reply:
since there's no word from Pat, and since it's his code, i'll resolve for now.